### PR TITLE
feat(TV UI): Player's Top controls redesign

### PR DESCRIPTION
**Pull request by KingLucius**
- Fixes Title alignment in case of RTL languages.
- Adds "Restart" & "Next Episode" in top controls bar.
- Removes the "Next Episode" button under progress seekbar.
- Changes "Tracks" icon.
- Margin for top bar is same on right & left.
- Removes the back button from player loading UI.

image

### CODE REVIEW

**Comment by fire-light42** [comment]
This might not be ready for review yet, but when pressing the reset button it might be nice to add a Y/N popup/popdown to make it impossible to make a missinput. I would also suggest that playerGoForward is always visible when hasNextEpisode because right now it hides itself when under 50%, this is due to mobile!=tv.

**Comment by KingLucius** [comment]
> This might not be ready for review yet, but when pressing the reset button it might be nice to add a Y/N popup/popdown to make it impossible to make a missinput. I would also suggest that playerGoForward is always visible when hasNextEpisode because right now it hides itself when under 50%, this is due to mobile!=tv.

- Do you mean Subtitles sync reset button ?
- playerGoForward is visible from the beginning not after 50% as depends on hasNextEpisode not PreloadNextEpisode

File: app/src/main/java/com/lagradost/cloudstream3/ui/player/GeneratorPlayer.kt [hunk 1]
```diff
@@ -1095,7 +1095,8 @@ class GeneratorPlayer : FullScreenPlayer() {
         }
 
         playerBinding?.playerSkipOp?.isVisible = isOpVisible
-        playerBinding?.playerSkipEpisode?.isVisible =
+
+        playerBinding?.playerGoForward?.isVisible =
```
Comment (fire-light42) [inline app/src/main/java/com/lagradost/cloudstream3/ui/player/GeneratorPlayer.kt:1099]: `playerGoForward is visible from the beginning not after 50% as it depends on hasNextEpisode not PreloadNextEpisode` 
No? `isOpVisible = percentage < SKIP_OP_VIDEO_PERCENTAGE` => `playerGoForward.isVisible = percentage > SKIP_OP_VIDEO_PERCENTAGE` meaning that in the event on an anime, the top button is not visible on TV. This was the behavior before on phone as we did not want to clutter the bottom bar, but on TV this should not be the behavior.

**Comment by fire-light42** [comment]
Any update on this? The UI looks nice from your screenshot.

**Comment by KingLucius** [comment]
> Any update on this? The UI looks nice from your screenshot.

Just got busy, will work on it next weekend

**Comment by KingLucius** [comment]
I was working on redesign for the player with logo, description & episode view but let's delay this as I am super busy these days
later I can resume work on the redesign

**Comment by DestroyerBDT** [comment]
So the back button will still be there

**Comment by DestroyerBDT** [comment]
Also, we don't need a restart button

**Comment by Luna712** [comment]
> Also, we don't need a restart button

@DestroyerBDT 

I disagree, on TV, reminding to the beginning when you want can be quite annoying.

**Review by fire-light42 — APPROVED** [review]
Code looks good, however I would like to have text under the icons when hovering like Netflix. 👍

**Comment by KingLucius** [comment]
> Code looks good, however I would like to have text under the icons when hovering like Netflix. 👍 

Ok will work on it

**Comment by KingLucius** [comment]
> Code looks good, however I would like to have text under the icons when hovering like Netflix. 👍

Done

Feels like this PR needs a lot of testings, don't have time for deep test these days so I need your help testing this

**Comment by fire-light42** [comment]
Why was this closed?

**Comment by KingLucius** [comment]
> Why was this closed?

The rebase ruined it for some reason, created a new one with additional fixes and will push today for review

**Pull request closed without merging**